wire-ios-sync-engine/Tests/Source/Integration/IntegrationTest.swift

```diff
@@ -46,7 +46,8 @@ final class MockAuthenticatedSessionFactory: AuthenticatedSessionFactory {
     override func session(
         for account: Account,
         coreDataStack: CoreDataStack,
-        configuration: ZMUserSession.Configuration = .init()
+        configuration: ZMUserSession.Configuration = .init(),
+        userDefaults: UserDefaults = .standard
```

Comment:
same here, just double checking standard is the right value since we know it could affect other tests

Reply:
fixed in c8a59a522c5b2a06d5a9bf871a2a7e174175247d